packages/card/src/Card.ts

```diff
@@ -149,9 +169,35 @@ export class Card extends ObserveSlotPresence(
         }
     }
 
+    private stopPropagationOnHref(event: Event): void {
+        if (this.href) {
+            event.stopPropagation();
+        }
+    }
+
+    private handlePointerdown(event: Event): void {
+        const path = event.composedPath();
+        const hasAnchor = path.some(
+            (el) => (el as HTMLElement).localName === 'a'
+        );
+        if (hasAnchor) return;
+        const start = +new Date();
+        const handleEnd = (): void => {
+            const end = +new Date();
+            if (end - start < 200) {
+                this.likeAnchor?.click();
```

Comment:
Should this also be `this.click()`?

Reply:
The card isn't actually a link, and when we `click` something we want it to be the actual link.